modin/core/storage_formats/pandas/parsers.py

```diff
@@ -152,7 +152,7 @@ def generic_parse(fname, **kwargs):
         num_splits = kwargs.pop("num_splits", None)
         start = kwargs.pop("start", None)
         end = kwargs.pop("end", None)
-        header_size = kwargs.pop("header_size", None)
+        header_size = kwargs.pop("header_size", 0)
```

Comment:
otherwise, we are getting `TypeError: 'NoneType' object cannot be interpreted as an integer` error